torch/csrc/api/include/torch/nn/options/activation.h

```diff
@@ -56,5 +57,15 @@ struct LeakyReLUOptions {
   TORCH_ARG(bool, inplace) = false;
 };
 
+// ============================================================================
+
+/// Options for the Softmax functional and module.
+struct TORCH_API SoftmaxOptions {
+  SoftmaxOptions(int dim);
+
+  // Dimension along which Softmax will be computed.
+  TORCH_ARG(int, dim);
```

Comment:
I think we should make `dim` have `int64_t` type, because that's the type of tensor dimensions for C++ tensors.

Reply:
OK, fixed it